docs/operating-scylla/nodetool-commands/tasks/index.rst

```diff
@@ -17,6 +17,16 @@ Nodetool tasks
 Task manager is an API-based tool for tracking long-running background operations, such as repair or compaction,
 which makes them observable and controllable. Task manager operates per node.
 
+Task Status Retention
+---------------------
+
+* When a task completes, its status is temporarily stored on the executing node
+* Status information is retained for up to :confval:`task_ttl_in_seconds` seconds
+* The status is automatically removed after being queried
```

Comment:
The status **of a finished task** is automatically removed after being queried **with task status or task tree**